### ISSUE TITLE
ok.ru

### ISSUE THREAD

**Issue by jungie1995** [vb6rocod/utils#43]
hello sir, can you update the ok.ru? thanks

**Comment by vb6rocod** [comment]
work for me.

**Comment by jungie1995** [comment]
yes its working for old videos. but for new on. it returns error 12 sir

**Comment by jungie1995** [comment]
https://ok.ru/video/5963859626731
can you test this out sir?

**Comment by racazahassem09191990** [comment]
It is working but only MP4 . it is good if with HLS

**Comment by vb6rocod** [comment]
add this line at end
$link=$z['hlsManifestUrl'];

**Comment by racazahassem09191990** [comment]
its not playable.

**Comment by vb6rocod** [comment]
use MPV.

**Comment by racazahassem09191990** [comment]
how to use it? im using ubuntu server with php streaming script.

**Comment by aptyumsudo** [comment]
how to set the 480P quality only not the higher quality for this project?

This is the mp4 quality list.

mobile = 144
lowest = 240
low = 360
sd = 480
hd = 720
full = 1080
